raven/models/base.py

```diff
@@ -542,8 +542,8 @@ def _assign_files(self, fns):
                                                    var_name=alt_name,
                                                    dimensions=ds[alt_name].dims,
                                                    units=ds[alt_name].attrs.get("units"),
+                                                   deaccumulate=ds[alt_name].attrs.get("GRIB_stepType", "") == "accum",
```

Comment:
This line makes it that I cannot force the "deaccumulate" flag in model.rvt.pr.deaccumulate=True, if the flag is not in the file it returns "" and overwrites my <True> tag. Up until here, the flag is set to true, but then gets overwritten here. So I guess the best alternative is to add that flag when I generate the new netcdf for Raven from the forecasts?

Reply:
Actually, I did just that, adding the GRIB_stepType attribute to 'accum' got it detected and works now.

Reply:
Hum, but ideally we would not overwrite it. I'll patch something.